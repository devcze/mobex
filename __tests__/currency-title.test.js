/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import CurrencyTitle from '../components/currency-title'

describe('CurrencyTitle', () => {
  it('CurrencyTitle render', () => {
    const component = renderer.create(<CurrencyTitle/>);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
