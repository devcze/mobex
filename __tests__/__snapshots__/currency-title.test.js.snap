// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrencyTitle CurrencyTitle render 1`] = `
<div
  className="currency-title__CurrencyStyle-sc-14ttzt-0 loBVpX"
/>
`;
